PREREQUISITES.md

```diff
@@ -397,7 +399,7 @@ problems.
 # <a name="troubleshooting"></a>Troubleshooting Installation
 - Verify your [environment variables](#environment) are set correctly and the
   paths exist
-
+- Developers are suggested for to use openssl1.1.1d version due to some technical issues.
```

Comment:
I would reword this to:

- Verify you are using OpenSSL 1.1.1d with `openssl version`